### PR TITLE
Adding text for -Content and -SourcePathorExtension parameters on *-Authenticode cmdlets

### LINKED ISSUE #1086: Get-AuthenticodeSignature is missing parameter descriptions and examples
For 5.1, Get-AuthenticodeSignature has Content and SourcePathOrExtension parameters. Content has no description; SourcePathOrExtension has a description, but it doesn't make sense. Neither of these parameters are represented in the examples. 

Hope this helps.

### PR DESCRIPTION
**Pull request by HemantMahawar**
Fixes #1086

Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

### CODE REVIEW

**Comment by zjalexander** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/5.1/Microsoft.PowerShell.Security/Get-AuthenticodeSignature.md](https://github.com/HemantMahawar/PowerShell-Docs/blob/Authenticode/reference/5.1/Microsoft.PowerShell.Security/Get-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/get-authenticodesignature?view=powershell-5.1&branch=pr-en-us-3232) (powershell-5.1) |
[reference/5.1/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md](https://github.com/HemantMahawar/PowerShell-Docs/blob/Authenticode/reference/5.1/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/set-authenticodesignature?view=powershell-5.1&branch=pr-en-us-3232) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Security/Get-AuthenticodeSignature.md](https://github.com/HemantMahawar/PowerShell-Docs/blob/Authenticode/reference/6/Microsoft.PowerShell.Security/Get-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/get-authenticodesignature?view=powershell-6&branch=pr-en-us-3232) (powershell-6) |
[reference/6/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md](https://github.com/HemantMahawar/PowerShell-Docs/blob/Authenticode/reference/6/Microsoft.PowerShell.Security/Set-AuthenticodeSignature.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.security/set-authenticodesignature?view=powershell-6&branch=pr-en-us-3232) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C2%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201811022053189288-3232%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=c865GRq2i01OQeBFdeVzDe5X%2Fh3CMWUTprZqiqaACEw%3D&st=2018-11-02T21%3A03%3A57Z&se=2018-12-03T21%3A08%3A57Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/5.1/Microsoft.PowerShell.Security/Get-AuthenticodeSignature.md [hunk 1]
```diff
@@ -11,66 +11,85 @@ title:  Get-AuthenticodeSignature
...
-PS C:\> Get-ChildItem $pshome\*.* | ForEach-object {Get-AuthenticodeSignature $_} | where {$_.status -eq "Valid"}
+### Example 3: Get only valid digital signatures for multiple files
+
+```powershell
+Get-ChildItem $pshome\*.* | ForEach-object {Get-AuthenticodeSignature $_} | Where-Object {$_.status -eq "Valid"}
 ```
 
-This command lists all of the files in the $pshome directory that have a valid Authenticode signature.
-The $pshome automatic variable contains the path to the Windows PowerShell installation directory.
+This command lists all of the files in the `$pshome` directory that have a valid digital signature.
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Security/Get-AuthenticodeSignature.md]: Change `$pshome` to `$PSHOME` to match the actual case of the variable.

File: reference/5.1/Microsoft.PowerShell.Security/Get-AuthenticodeSignature.md [hunk 2]
```diff
@@ -11,66 +11,85 @@ title:  Get-AuthenticodeSignature
...
+The `$pshome` automatic variable contains the path to the Windows PowerShell installation directory.
 
-The command uses the **Get-ChildItem** cmdlet to get the files in the $pshome directory.
+The command uses the **Get-ChildItem** cmdlet to get the files in the `$pshome` directory.
```
Comment (sdwheeler) [inline reference/5.1/Microsoft.PowerShell.Security/Get-AuthenticodeSignature.md]: Change $pshome to $PSHOME to match the actual case of the variable.

File: reference/6/Microsoft.PowerShell.Security/Get-AuthenticodeSignature.md [hunk 3]
```diff
@@ -11,65 +11,100 @@ title:  Get-AuthenticodeSignature
...
-PS C:\> Get-ChildItem $pshome\*.* | ForEach-object {Get-AuthenticodeSignature $_} | where {$_.status -eq "Valid"}
+### Example 3: Get only valid digital signatures for multiple files
+
+```powershell
+Get-ChildItem $pshome\*.* | ForEach-object {Get-AuthenticodeSignature $_} | Where-Object {$_.status -eq "Valid"}
 ```
 
-This command lists all of the files in the $pshome directory that have a valid Authenticode signature.
-The $pshome automatic variable contains the path to the PowerShell installation directory.
+This command lists all of the files in the `$pshome` directory that have a valid digital signature.
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Security/Get-AuthenticodeSignature.md]: Change $pshome to $PSHOME to match the actual case of the variable.

**Pull request merged**